docs/modules/opa/pages/usage-guide/user-info-fetcher.adoc

```diff
@@ -46,21 +46,54 @@ stringData:
 ----
 
 <1> Enable the `user-info-fetcher` sidecar
-<2> Enable TLS verification using the CA from the `tls` secretClass.
+<2> Enable TLS verification using the CA from the `tls` SecretClass.
 <3> Obtain Keycloak API credentials from the specified secret. The Secret must have `clientId` and `clientSecret` entries.
 <4> Refer to the applicable realm in your Keycloak server.
 
 Currently the following backends are supported:
 
-* <<_keycloak>>
+* xref:#backend-keycloak[]
+* xref:#backend-activedirectory[]
 
+[#backend-keycloak]
```

Comment:
I know it was this way before, but IMHO it makes sense to move the Keycloak example above below the Keycloak section :)

Reply:
The API docs are fairly independent from the backend in use, mentioned that the example shows some keycloakisms in db442967e6b29d320c8fd66997b615c7ab8551dc

Reply:
Thanks